package.json

```diff
@@ -44,5 +46,10 @@
     "react-router": "^2.4.0",
     "sinon": "^1.17.2",
     "sinon-chai": "^2.8.0"
+  },
+  "publishr": {
+    "dependencies": [
+      "^builder"
+    ]
```

Comment:
If you want to exclude the source code from npm, you will want to add:

```js
"files": {
  ".npmignore": ".npmignore.publishr"
}
```

Then add a `.npmignore.publishr` file containing:

```
/src
```

Add anything else you want to exclude from npm in that file. The idea here is that installing from a git repo actually obeys the `.npmignore` file, so you can't actually check it into the repo. That is why you check in a file named `.npmignore.publishr` instead and have `publishr` sort out the mess.